### PR TITLE
profiles: Update to header 269

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: scripts/gen_profiles_solution.py [hunk 1]
```diff
@@ -2233,6 +2233,9 @@ def applyWorkarounds(self):
         if 'VkPhysicalDeviceFragmentDensityMapOffsetPropertiesQCOM' in self.structs:
             self.structs['VkPhysicalDeviceFragmentDensityMapOffsetPropertiesQCOM'].members['fragmentDensityOffsetGranularity'].limittype = 'min,mul'
 
+        if 'VkPhysicalDeviceSchedulingControlsPropertiesARM' in self.structs:
+            self.structs['VkPhysicalDeviceSchedulingControlsPropertiesARM'].members['schedulingControlsFlags'].limittype = 'min'
```
Comment (ziga-lunarg) [inline scripts/gen_profiles_solution.py:2237]: @christophe-lunarg Is this correct?

Comment (ziga-lunarg) [reply scripts/gen_profiles_solution.py:2237]: If I don't add it I'm getting:
`Unsupported limittype 'None' in member 'schedulingControlsFlags' of structure 'VkPhysicalDeviceSchedulingControlsPropertiesARM'`

Comment (christophe-lunarg) [reply scripts/gen_profiles_solution.py:2237]: Pulling...

Comment (christophe-lunarg) [reply scripts/gen_profiles_solution.py:2237]:
```
        if 'VkPhysicalDeviceSchedulingControlsPropertiesARM' in self.structs:
            self.structs['VkPhysicalDeviceSchedulingControlsPropertiesARM'].members['schedulingControlsFlags'].limittype = 'bitmask'
```
Fixed it for me :)

Comment (ziga-lunarg) [reply scripts/gen_profiles_solution.py:2237]: Thanks!

**Review by christophe-lunarg — APPROVED** [review]

**Pull request merged**